src/components/PeoplePage.tsx

```diff
@@ -1,8 +1,77 @@
-import { PeopleFilters } from './PeopleFilters';
+import { useEffect, useMemo, useState } from 'react';
+import { useSearchParams } from 'react-router-dom';
+import { PeopleFilters } from './PeopleFilters/PeopleFilters';
 import { Loader } from './Loader';
 import { PeopleTable } from './PeopleTable';
+import { Person } from '../types';
+import { getPeople } from '../api';
+import { FilteredPeople } from './FilteredPeople';
```

Comment:
Consider renaming 'FilteredPeople' to 'filterPeople' to indicate it's a function, not a component.